images_test.go

```diff
@@ -0,0 +1,395 @@
+package cloudflare
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func timeMustParse(layout, value string) time.Time {
+	t, err := time.Parse(layout, value)
+	if err != nil {
+		panic(err)
+	}
+	return t
+}
+
+var expectedImageStruct = Image{
+	ID:       "ZxR0pLaXRldlBtaFhhO2FiZGVnaA",
+	Filename: "avatar.png",
+	Metadata: map[string]interface{}{
+		"meta": "metaID",
+	},
+	RequireSignedURLs: true,
+	Variants: []string{
+		"https://imagedelivery.net/MTt4OTd0b0w5aj/ZxR0pLaXRldlBtaFhhO2FiZGVnaA/hero",
+		"https://imagedelivery.net/MTt4OTd0b0w5aj/ZxR0pLaXRldlBtaFhhO2FiZGVnaA/original",
+		"https://imagedelivery.net/MTt4OTd0b0w5aj/ZxR0pLaXRldlBtaFhhO2FiZGVnaA/thumbnail",
+	},
+	Uploaded: timeMustParse(time.RFC3339, "2014-01-02T02:20:00Z"),
+}
+
+func TestUploadImage(t *testing.T) {
+	setup(UsingAccount("foo"))
```

Comment:
let's use the `testAccountID` value here instead of "foo".

```suggestion
	setup(UsingAccount(testAccountID))
```